### PR TITLE
Add aomic masks

### LINKED ISSUE #177: [ENH]: Add the fMRIprep provided brain mask patterns in AOMIC datagrabbers
### Are you requiring a new dataset or marker?

- [X] I understand this is not a marker or dataset request

### Which feature do you want to include?

fMRIprep outputs brain masks for each element/datatype and these are *typically* more liberal than the nilearn computed masks. In most cases liberal is better for preprocessing. These can be added to the datagrabber to be accessed as a type.

### How do you imagine this integrated in junifer?

I can implement it for the AOMIC datagrabber as discussed in #174

### Do you have a sample code that implements this outside of junifer?

_No response_

### Anything else to say?

_No response_

### PR DESCRIPTION
**Pull request by LeSasse**
* [x] fix #177
* [x] description of feature/fix
* [x] tests added/passed
* [x] add an entry to the [latest changes](../docs/changes/latest.inc)

This pull request adds the fMRIprep masks for BOLD images in MNI space as well as T1w images in MNI space to the AOMIC datagrabbers.

### CODE REVIEW

**Comment by LeSasse** [comment]
@synchon any idea why tox tests are failing for some of these?

**Comment by synchon** [comment]
> @synchon any idea why tox tests are failing for some of these?

HCP data fetching issues with datalad. You might wanna rebase on `main` just to be sure you have the latest HCP test changes.

**Comment by LeSasse** [comment]
> > @synchon any idea why tox tests are failing for some of these?
> 
> HCP data fetching issues with datalad. You might wanna rebase on `main` just to be sure you have the latest HCP test changes.

thanks, you should make yourself a little Github bot that always replies for people to rebase on main.

**Comment by synchon** [comment]
> > > @synchon any idea why tox tests are failing for some of these?
> > 
> > 
> > HCP data fetching issues with datalad. You might wanna rebase on `main` just to be sure you have the latest HCP test changes.
> 
> thanks, you should make yourself a little Github bot that always replies for people to rebase on main.

lol, yes totally, weekend to-do.

File: docs/changes/latest.inc [hunk 1]
```diff
@@ -40,6 +40,9 @@ Enhancements
 - Add support for nilearn computed masks (``compute_epi_mask``, ``compute_brain_mask``, ``compute_background_mask``,
   ``fetch_icbm152_brain_gm_mask``) (:gh:`175` by `Fede Raimondo`_).
 
+- Add fMRIprep brain masks to the datagrabber patterns for all datagrabbers in the aomic sub-package
```
Comment (synchon) [inline docs/changes/latest.inc:43]: Just to follow the naming convention: Add fMRIPrep ...

File: docs/changes/latest.inc [hunk 2]
```diff
@@ -40,6 +40,9 @@ Enhancements
...
+  (:gh:`177` by `Leonard Sasse`_)
```
Comment (synchon) [inline docs/changes/latest.inc:44]: `.` at the end.

**Review by synchon — CHANGES_REQUESTED** [review]

File: docs/changes/latest.inc (hunk 1 above)
Comment (LeSasse) [reply docs/changes/latest.inc:43]: oops yeah always get that wrong

**Review by synchon — APPROVED** [review]

**Pull request merged**